Mathlib/Analysis/Analytic/Polynomial.lean

```diff
@@ -74,4 +74,8 @@ theorem AnalyticOn.eval_linearMap' (f : σ → E →ₗ[𝕜] B) (p : MvPolynomi
 theorem AnalyticOn.eval_mvPolynomial [Fintype σ] (p : MvPolynomial σ 𝕜) :
     AnalyticOn 𝕜 (eval · p) Set.univ := AnalyticOn.eval_linearMap (.id (R := 𝕜) (M := σ → 𝕜)) p
 
+theorem MvPolynomial.continuous_eval (p : MvPolynomial ι ℝ) :
+    Continuous fun x ↦ (eval x) p := by
+  continuity
```

Comment:
This holds for any [TopologicalSemiring](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Ring/Basic.html#TopologicalSemiring) in place of ℝ using [MvPolynomial.induction_on](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/MvPolynomial/Basic.html#MvPolynomial.induction_on)

Also, there's already a σ around, so I think you don't need to add ι to the `variable`.